### DISCUSSION TITLE
Will move repo to organization from personal account

### DISCUSSION THREAD

**Discussion by sp-yduck** [k8s-proxmox/cluster-api-provider-proxmox#154] [General]
Hi folks.
Now cluster-api-provider-proxmox is almost reaching to 50 stars⭐🎉!!!, I am feeling the project is getting bigger and would like to involve more people to growth the project. So, I am thinking to move this repo to organization from my personal account.
New location would be: https://github.com/k8s-proxmox. I belive I won't mess up and the repo will be properly migrated to new location...
Please note that once I transfer the repo to organization, `https://github.com/sp-yduck/cluster-api-provider-proxmox` this url will be my folked repository and not canonical repository.
Thank you for the contribution you all so far !

**Comment by UnstoppableMango** [comment]
Excited to see this project grow, I love what you're doing!
